ctrd/client.go

```diff
@@ -32,8 +32,6 @@ type Client struct {
 	watch *watch
 	lock  *containerLock
 
-	rpcAddr string
```

Comment:
I think the field is used:
```
// WithRPCAddr set containerd listen address.
func WithRPCAddr(rpcAddr string) ClientOpt {
	return func(c *clientOpts) error {
		if rpcAddr == "" {
			return fmt.Errorf("rpc socket path is empty")
		}

		c.rpcAddr = rpcAddr
		return nil
	}
}
```

Please help confirm this. Thanks. @fuweid

Reply:
looks good to remove the field.